src/task 33/main.js

```diff
@@ -0,0 +1,15 @@
+var butt = document.getElementById("button");
+var tArea = document.getElementById("code");
+var pre = document.getElementById("output")
+
+
+function workbench(){
```

Comment:
Final change - please better pass all document node via function arguments. Otherwise your function can't be reusable for other set of fields.. It will anyway refer to the same set of nodes.